fairscale/optim/utils.py

```diff
@@ -44,3 +46,31 @@ def recursive_copy_to_device(value: Any, non_blocking: bool, device: torch.devic
         return device_val
 
     return value
+
+
+def broadcast_object(
+    obj: Any, src_rank: int, group: object = dist.group.WORLD, dist_device: torch.device = torch.device("cpu")
```

Comment:
add a comment on why is this function needed here?

Reply:
ah yes, I added comments where I was pulling that in, it's not in the old torch.. adding another comment here